srcpkgs/iputils/template

```diff
@@ -1,39 +1,29 @@
 # Template file for 'iputils'
 pkgname=iputils
-version=20180629
-revision=3
-patch_args="-Np1"
+version=20190515
+revision=1
 wrksrc="${pkgname}-s${version}"
+build_style=meson
+configure_args="-DNO_SETCAP_OR_SUID=true -DUSE_IDN=false
+ -DBUILD_TFTPD=false -DBUILD_NINFOD=false"
+hostmakedepends="pkg-config docbook2x docbook-xsl-ns"
 makedepends="libcap-devel libressl-devel"
 depends="libcap-progs"
 short_desc="Useful utilities for Linux networking (including ping)"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
 license="BSD-3-Clause, GPL-2.0-or-later"
 homepage="https://github.com/iputils/iputils"
-distfiles="https://github.com/${pkgname}/${pkgname}/archive/s${version}.tar.gz"
-checksum=da14105291dd491f28ea91ade854ed10aee8ba019641c80eed233de3908be7c5
+distfiles="https://github.com/iputils/iputils/archive/s${version}.tar.gz"
+checksum=9b5125eb5ef9f4e947ad8fdddcf77f538f53b8f47b53eb5bc5347cb16d01c8fd
 
 alternatives="
  ping:ping:/usr/bin/${pkgname}-ping
  ping:ping.8:/usr/share/man/man8/${pkgname}-ping.8
  ping6:ping6:/usr/bin/${pkgname}-ping
  ping6:ping6.8:/usr/share/man/man8/${pkgname}-ping.8"
```

Comment:
ping6 alternative still exists... I think it's redundant because ping(8) has -4 and -6 options.

Reply:
Not sure what the problem is here. ping6 is pointing to iputils-ping... do we no longer wish to provide a ping6 compat link?

Reply:
IMHO this ping6 symlink is just redundant but I don't care enough to be opposed to keep it.